tomitribe-crest/src/main/java/org/tomitribe/crest/cmds/CmdMethod.java

```diff
@@ -237,6 +237,10 @@ public Object convert(final Arguments arguments, final Needed needed) {
 
             final List<Object> converted = CmdMethod.this.convert(arguments, needed, parameters);
 
+            if (allValuesAreNull(converted)) {
```

Comment:
shouldnt we have a @Options(nullible = true)? Fear it will break consumers to do it now.